client/lib/RoomManager.coffee

```diff
@@ -105,16 +105,14 @@ Meteor.startup ->
 		return room?.dom?
 
 	updateUserStatus = (user, status, utcOffset) ->
-		onlineUsersValue = onlineUsers.curValue
+		allUsersValue = allUsers.curValue
+		allUsersValue[user.username] =
+			firstName: user.profile.first_name
+			lastName: user.profile.last_name
+			name: user.name
+			status: user.status
```

Comment:
This isn't a bug here, but rather in how we're populating our users collection from LDAP. We're not populating a 'status' field on startup, so only users that have logged in before have it set.

The list of users on the right pane will use this field to determine how to style the name and the dot next to it (e.g., green == available, red == away). Normally, if a user is offline, it will appear as gray, making it obvious that the user is offline. However, in our case, if a user hasn't logged in before, the field is missing, so the name shows up in black with no dot.

I'll file an actual bug ticket about this. If we want, we can also work around it here by setting 'status' to:
```
status: if user.status? then user.status else 'offline'
```
but probably isn't worth it. Thoughts?